cmd/erasure-server-pool-decom.go

```diff
@@ -496,7 +496,7 @@ func (z *erasureServerPools) Init(ctx context.Context) error {
 	}
 
 	// Start rebalance routine
-	z.StartRebalance()
+	z.StartRebalance(true)
```

Comment:
Can we think of using global context to have this flag for init? Does that make even sense?